Sort screenshots chronologically in descending order

QazCetelic opened the pull request:
Feature request as mentioned in #2763.

# Test

Screenshots used for test and their modification times
```
Mar 20 21:25 'Mox Landing Cropped.png'
Mar 22 17:07 "97b's platform.png"
Dec  3  2023  2023-12-03_21.37.22.png
Jun 23  2023  2023-06-23_15.29.04.png
Jun  6  2023  2023-06-06_22.05.34.png
```

Before:
![before_image](https://github.com/user-attachments/assets/66489539-e514-47ff-98df-457b813682db)


After:
![after_image](https://github.com/user-attachments/assets/e97646c5-0106-4ce4-a88a-672029b7056e)

Commit:
Sort files chronologically in descending order

Signed-off-by: Qaz <dev@malplena.net>

TheKodeToad reviewed (approved):
In future it would be nice to have a combobox to change the sort mode, but I agree this is a better default

Trial97 commented:
I will approve once https://github.com/PrismLauncher/PrismLauncher/pull/2798/checks?check_run_id=29876894685 passes

Commit:
DCO Remediation Commit for QazCetelic <qaz.cetelic@protonmail.com>

I, QazCetelic <qaz.cetelic@protonmail.com>, hereby add my Signed-off-by to this commit: 7fbf0d3bf3c3821916119a5303d6bc302f954580

Signed-off-by: QazCetelic <qaz.cetelic@protonmail.com>
